tools/reviewer/web/src/app/dashboard/reviews-panel/review/editable-property/editable-property.component.html

```diff
@@ -50,7 +50,8 @@
       <mat-form-field fxFlex 
         *ngIf="property === 'description'">
         <textarea matInput placeholder=""
-          matTextareaAutosize 
+          matTextareaAutosize
+          matAutosizeMaxRows="8" 
```

Comment:
It's a tough one because it doesn't has that property. Removing `matTextareaAutosize` results in smaller textarea when starting to type and if we remove min or max rows then the textarea behaves weird as well. To make it less complicated we could move to a simple HTML textarea, that's what I did in replypopup.

Reply:
Let's keep it for a next PR. Probably in designs most of the material stuff is going away.